doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -15,26 +15,71 @@ xmlns:utu="using:Uno.Toolkit.UI"
        BasedOn="{StaticResource DrawerFlyoutPresenterStyle}"
        TargetType="FlyoutPresenter">
     <Setter Property="utu:DrawerFlyoutPresenter.OpenDirection" Value="Top" />
+    <Setter Property="utu:DrawerFlyoutPresenter.DrawerDepth" Value="0.66*" />
     <Setter Property="utu:DrawerFlyoutPresenter.LightDismissOverlayBackground" Value="#80808080" />
     <Setter Property="utu:DrawerFlyoutPresenter.IsGestureEnabled" Value="True" />
 </Style>
 <!-- and/or -->
 <utu:DrawerFlyoutPresenter OpenDirection="Top"
+                           DrawerDepth="0.66*"
                            LightDismissOverlayBackground="#80808080"
                            IsGestureEnabled="True" />
 ```
 
+> note:
```

Comment:
Is this meant to show as an alert on DocFX? If so, we should set it as `[!NOTE]` on its own line, maybe here we use a WARN?

[docfx usage](https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet#alerts)